src/Tracy/Dumper/Dumper.php

```diff
@@ -501,7 +502,7 @@ public static function encodeString(string $s, int $maxLength = null): string
 		static $table;
 		if ($table === null) {
 			foreach (array_merge(range("\x00", "\x1F"), range("\x7F", "\xFF")) as $ch) {
-				$table[$ch] = '\x' . str_pad(dechex(ord($ch)), 2, '0', STR_PAD_LEFT);
+				$table[$ch] = '\x' . str_pad(dechex(ord((string) $ch)), 2, '0', STR_PAD_LEFT);
```

Comment:
> Parameter #1 $character of function ord expects string, float|int given.

Not true, it's a bug in phpstan. Should be fixed in upstream and ignored here.

Reply:
Reverted.